specification/src/main/asciidoc/platform/TransactionManagement.adoc

```diff
@@ -278,37 +277,36 @@ finally, commit the transaction.
 
 === Transaction Interoperability
 
-=== Multiple Java EE Platform Interoperability
+=== Multiple Jakarta EE Platform Interoperability
 
 This specification does not require the Product
 Provider to implement any particular protocol for transaction
-interoperability across multiple Java EE products. Java EE compatibility
-requires neither interoperability among identical Java EE products from
-the same Product Provider, nor among heterogeneous Java EE products from
+interoperability across multiple Jakarta EE products. Jakarta EE compatibility
+requires neither interoperability among identical Jakarta EE products from
+the same Product Provider, nor among heterogeneous Jakarta EE products from
 multiple Product Providers.
 
-We recommend that Java EE Product Providers use
+We recommend that Jakarta EE Product Providers use
 the IIOP transaction propagation protocol defined by OMG and described
-in the OTS specification (and implemented by the Java Transaction
-Service), for transaction interoperability when using the EJB
+in the OTS specification (and implemented by the Jakarta Transactions
+Service), for transaction interoperability when using the Jakarta Enterprise Beans
 interoperability protocol based on RMI-IIOP.
 
 === Support for Transactional Resource Managers
 
-This specification requires all Java EE
+This specification requires all Jakarta EE
 products to support the _javax.transaction.xa.XAResource_ interface, as
 specified in the Connector specification. This specification also
-requires all Java EE products to support the
+requires all Jakarta EE products to support the
 _javax.transaction.xa.XAResource_ interface for performing two-phase
-commit operations on JDBC drivers that support the JTA XA APIs. This
-specification does not require that JDBC drivers or JMS providers use
+commit operations on JDBC drivers that support the Jakarta Transactions XA APIs. This
```

Comment:
I think XAResource is part of Java SE now? https://docs.oracle.com/javase/8/docs/api/javax/transaction/xa/XAResource.html

Reply:
Yes, the JTA XA APIs are not part of the Jakarta Transactions spec.